packages/e2e-tests/specs/editor/blocks/classic.test.js

```diff
@@ -98,4 +101,45 @@ describe( 'Classic', () => {
 		await page.waitForSelector( '.wp-block[data-type="core/gallery"]' );
 		expect( await getEditedPostContent() ).toMatch( /<!-- wp:gallery/ );
 	} );
+
+	it( 'Should not fail after save/reload', async () => {
+		const mainBlockCSSSelector = 'div[aria-label^="Block: Classic"]';
+
+		const clickClassic = async () => {
+			await page.waitForSelector( mainBlockCSSSelector );
+			await page.click( mainBlockCSSSelector );
+		};
+
+		// Might move to utils if this becomes useful enough for other tests
+		const runWithoutCache = async ( cb ) => {
+			await page.setCacheEnabled( false );
+			await cb();
+			await page.setCacheEnabled( true );
+		};
+
+		await insertBlock( 'Classic' );
+
+		// Wait for TinyMCE to initialise.
+		await page.waitForSelector( '.mce-content-body' );
+
+		// Ensure there is focus.
+		await page.focus( '.mce-content-body' );
+		await page.keyboard.type( 'test' );
+
+		// Move focus away.
+		await pressKeyWithModifier( 'shift', 'Tab' );
+
+		// Save
+		await saveDraft();
+
+		// Reload
+		// Disabling the browser disk cache is needed in order to reproduce the issue
+		// in case it regresses. To test this, revert commit 65c9f74, build and run the test.
+		await runWithoutCache( async () => {
+			await page.reload();
+			await clickClassic();
+			expect( console ).not.toHaveErrored();
+			expect( await getEditedPostContent() ).toMatchSnapshot();
+		} );
```

Comment:
Do all of these need to be inside of `runWithoutCache`, or would just `await page.reload();` suffice?

```suggestion
		} );
		await clickClassic();
		expect( console ).not.toHaveErrored();
		expect( await getEditedPostContent() ).toMatchSnapshot();
```

Reply:
Good point, I think only `page.reload()` is needed there. 👍🏻